examples/xonotic/Makefile

```
@@ -23,12 +23,41 @@
 #     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
 #
 
-REPOSITORY = us-docker.pkg.dev/agones-images/examples
+REPOSITORY ?= us-docker.pkg.dev/agones-images/examples
+
+WINDOWS_VERSIONS = ltsc2019 ltsc2022
+BUILDX_WINDOWS_BUILDER = windows-builder
+
+# Build with Windows support
+WITH_WINDOWS ?= 1
+# Build with ARM64 support
+WITH_ARM64 ?= 1
+
+# When pushing set to --push
+WINDOWS_DOCKER_PUSH_ARGS = --push
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
+server_tag = $(REPOSITORY)/xonotic-example:1.2
+
+ifeq ($(WITH_WINDOWS), 1)
+server_tag_linux_amd64 = $(server_tag)-linux-amd64
+server_tag_linux_arm64 = $(server_tag)-linux-arm64
+else
+server_tag_linux_amd64 = $(server_tag)-amd64
+server_tag_linux_arm64 = $(server_tag)-arm64
+endif
```

```suggestion
server_tag_linux_amd64 = $(server_tag)-linux-amd64
server_tag_linux_arm64 = $(server_tag)-linux-arm64
```

I know we do this everywhere, but I don't know why, so let's break the habit and remove the conditional. It's not really doing anything.

---

> I know we do this everywhere, but I don't know why, so let's break the habit and remove the conditional. It's not really doing anything.

Even, I was confused why we are doing this everywhere. But, At the end I had to follow the coding guidelines (Current codebase written in this way)

---

> 
server_tag_linux_arm64 = $(server_tag)-linux-arm64
I think, this is not required. Since, Xonotic does not have support for ARM64, I am removing everthing related to ARM64 in makefile.